.coveralls.yml

```diff
@@ -1,2 +1 @@
-service_name: travis-pro
-repo_token: S5wTeingaGKIID1VojIa9l4McqoCl4YAi
\ No newline at end of file
+repo_token: process.env.COVERALLS_REPO_TOKEN
```

Comment:
This is not part of your work. This is already merged. Please rebase using the current state of develop.